src/model/BoxPlotColumn.ts

```diff
@@ -0,0 +1,141 @@
+/**
+ * Created by bikramkawan on 24/11/2016.
+ */
+import * as d3 from 'd3';
+import ValueColumn from './ValueColumn';
+import Column from './Column';
+import {IValueColumnDesc} from './ValueColumn';
+import {numSort} from './MultiValueColumn';
+
+export enum Sort {
+
+  min, max, median, q1, q3, mean
+}
+
+
+export interface IBoxPlotColumn {
+  getBoxPlotData(row: any, index: number): IBoxPlotData;
+  getDataInfo(): IBoxPlotColumnDesc;
+  getUserSortBy(): string;
```

Comment:
you could rename this method to `getSortMethod` would better indicate the purpose of the method